connectors/commons/src/main/java/com/datastax/oss/dsbulk/connectors/commons/AbstractFileBasedConnector.java

```diff
@@ -566,4 +567,8 @@ protected URL getOrCreateDestinationURL() {
     // assume we are writing to a single URL and ignore fileNameFormat
     return urls.get(0);
   }
+
+  protected boolean isDataSizeSamplingAvailable() {
+    return !(urls != null && urls.size() == 1 && urls.get(0).getProtocol().equals(STD_PROTOCOL));
```

Comment:
I'm having second thoughts about this impl because it is possible (although weird) to specify a `std:/` URL in a url file, and it wouldn't be detected here.